### ISSUE TITLE
wsdl2py jinja2 template not found

### ISSUE THREAD

**Issue by vpistis** [soapteam/soapfish#98]
Hi,

the command: `wsdl2py --client mysoap.wsdl` return this error:
```python
Traceback (most recent call last):
  File "/Users/user/.virtualenvs/soapfish/bin/wsdl2py", line 11, in <module>
    load_entry_point('soapfish==0.6.0.dev0', 'console_scripts', 'wsdl2py')()
  File "/Users/user/.virtualenvs/soapfish/lib/python3.4/site-packages/soapfish/wsdl2py.py", line 144, in main
    code = generate_code_from_wsdl(xml, opt.target, opt.use_wsa, cwd=cwd)
  File "/Users/user/.virtualenvs/soapfish/lib/python3.4/site-packages/soapfish/wsdl2py.py", line 101, in generate_code_from_wsdl
    schemas = ''.join(schema_to_py(schema, xsd_namespaces, **kw) for schema in schemas)
  File "/Users/user/.virtualenvs/soapfish/lib/python3.4/site-packages/soapfish/wsdl2py.py", line 101, in <genexpr>
    schemas = ''.join(schema_to_py(schema, xsd_namespaces, **kw) for schema in schemas)
  File "/Users/user/.virtualenvs/soapfish/lib/python3.4/site-packages/soapfish/xsd2py.py", line 166, in schema_to_py
    tpl = env.get_template('xsd')
  File "/Users/user/.virtualenvs/soapfish/lib/python3.4/site-packages/jinja2/environment.py", line 830, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "/Users/user/.virtualenvs/soapfish/lib/python3.4/site-packages/jinja2/environment.py", line 804, in _load_template
    template = self.loader.load(self, name, globals)
  File "/Users/user/.virtualenvs/soapfish/lib/python3.4/site-packages/jinja2/loaders.py", line 113, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "/Users/user/.virtualenvs/soapfish/lib/python3.4/site-packages/jinja2/loaders.py", line 235, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: xsd
```

Any idea?

Thanx :)

**Comment by techdragon** [comment]
#99 Should fix this.

**Comment by pope1ni** [comment]
Fixed by cde510b.

**Comment by vpistis** [comment]
thanks a lot, I try it in these days
